src/App.tsx

```diff
@@ -1,31 +1,63 @@
 import React from 'react';
 import './App.scss';
+import { getTodos } from './components/api';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
 
 interface State {
   selectedUserId: number;
+  todos: [];
```

Comment:
```suggestion
  todos: Task[];
```
Давай використаємо інтерфейс Task